src/js/outreach/views/opt-in_views.js

```diff
@@ -51,6 +51,9 @@ const OptInView = View.extend({
     <p class="opt-in__disclaimer">By clicking Submit you agree to receive SMS text message notifications. You may opt out at any time.</p>
     <button class="opt-in__submit button--green w-100 js-submit" disabled>Submit</button>
   `,
+  modelEvents: {
+    'change': 'setSubmitButtonState',
```

Comment:
`setSubmitButtonState` should not use `this.ui.foo.val()` and should instead us `this.model.isValid()`. The data on the model is not necessarily the same data that's in the form.. `this.ui.phone.val` specifically.

Reply:
This has been updated in my latest push.